scripts/migrate-to-disabled-models.js

```diff
@@ -0,0 +1,98 @@
+import { eq } from "drizzle-orm";
+import { drizzle } from "drizzle-orm/node-postgres";
+import { pgTable, text } from "drizzle-orm/pg-core";
+// required for local:
+// import dotenv from "dotenv";
+// dotenv.config();
+
+// run with: npm run migrate-to-disabled-models
+// this will populate column disabled_models for all tenants
+
+const databaseUrl = process.env.DATABASE_URL;
+if (!databaseUrl) throw new Error("DATABASE_URL environment variable is required");
+
+const ALL_VALID_MODELS = [
+  "claude-sonnet-4-20250514",
+  "gpt-4o",
+  "gpt-3.5-turbo",
+  "gpt-4.1-2025-04-14",
+  "o3-2025-04-16",
+  "gemini-2.0-flash",
+  "gemini-1.5-pro",
+  "meta-llama/llama-4-scout-17b-16e-instruct",
+  "claude-3-7-sonnet-latest",
+  "claude-3-5-haiku-latest",
+  "claude-opus-4-20250514",
+];
+
+function showUsage() {
+  console.log("Usage: npm run migrate-to-disabled-models");
+  process.exit(1);
+}
+
+if (process.argv.length > 2) {
+  console.log("Error: Arguments provided");
+  showUsage();
+}
+
+const db = drizzle(databaseUrl);
+
+const tenantsSchema = pgTable("tenants", {
+  id: text("id").primaryKey(),
+  slug: text("slug").notNull(),
+  enabledModels: text("enabled_models").array(),
+  disabledModels: text("disabled_models").array(),
+});
+
+function getDisabledModelsFromEnabled(enabledModels) {
+  if (enabledModels === null) {
+    return [];
+  }
+
+  return ALL_VALID_MODELS.filter((model) => !enabledModels.includes(model));
+}
+
+console.log("Migrating to disabled_models approach...");
```

Comment:
Nit: Can we put all the functions together and the code that runs together?

It's harder to follow the logic when I see a function definition in the middle of it.